Modules/IO/NIFTI/test/itkNiftiImageIOTest14.cxx

```diff
@@ -0,0 +1,104 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkNiftiImageIOTest.h"
+#include "itkTestingMacros.h"
+
+// Test writing NIFTI xyzt_units field - should be 10 (mm, sec)
+int
+itkNiftiImageIOTest14(int argc, char * argv[])
+{
+
+  if (argc > 1)
+  {
+    char * testdir = *++argv;
+    itksys::SystemTools::ChangeDirectory(testdir);
```

Comment:
FWIW, I would rather just do `testdir = argv[1]`, without the pointer increment (`++argv`).

Reply:
Thanks I will change this, also don't need the following `success` variable